tests/support/loadJsdom.ts

```diff
@@ -31,6 +31,13 @@ global.requestAnimationFrame = (cb: (...args: any[]) => {}) => {
 
 global.cancelAnimationFrame = () => {};
 
+global.IntersectionObserver = function () {
```

Comment:
That would be nice, but would need to live in `src` so that it could be used externally...